organization/eu-central-1/app-runner/config/terragrunt.hcl

```
@@ -0,0 +1,39 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+
+include "env" {
+  path = "${get_terragrunt_dir()}/../../../_common/app_runner.hcl"
+}
+
+dependency "auto_scaling" {
+  config_path = "../auto_scaling"
+}
+
+inputs = {
+
+  service_name                     = "remote-planning-poker"
+  create_custom_domain_association = true
+  domain_name                      = "estimation.xaynet.dev"
+  enable_www_subdomain             = true
+  auto_scaling_configuration_arn   = dependency.auto_scaling.outputs.auto_scaling_configurations["default"].arn
```

one small thing:
```suggestion
  auto_scaling_configuration_arn   = dependency.auto_scaling.outputs.auto_scaling_configurations["default"].arn
  
  enable_observability_configuration = false
```
just to be sure that it will not create any costs in the future